src/models/UserType.ts

```diff
@@ -0,0 +1,8 @@
+export enum UserType {
+  GENERAL = 'General',
+  BREEDER = 'Breeder',
+  ADOPTIONCENTRE = 'Adoption Centre',
```

Comment:
minor: We usually use underscores to separate words in constants, so this would be...
```
...
ADOPTION_CENTRE = 'Adoption Centre',
...
```